Enable logind plugin when -Delogind=true is set as well

--- fwupd-1.3.9/plugins/meson.build
+++ fwupd-1.3.9/plugins/meson.build
@@ -27,7 +27,7 @@
 subdir('wacom-raw')
 endif
 
-if get_option('systemd')
+if get_option('systemd') or get_option('elogind')
 subdir('logind')
 endif
 
